packages/web/tailwind.config.js

```diff
@@ -12,11 +12,10 @@ module.exports = {
     ...createGlobPatternsForDependencies(__dirname),
   ],
   theme: {
-    extend: {
-      fontFamily: {
-        hebrew: ['"SBL-Hebrew"', '"Times New Roman"', 'serif'],
-        greek: ['"SBL-Greek"', '"Times New Roman"', 'serif'],
-      },
+    fontFamily: {
+      sans: '"Noto Sans"',
```

Comment:
We should add a fallback fonts for while our font is still loading. Here is how tailwind recommends doing that: https://tailwindcss.com/docs/font-family#customizing-the-default-font